_episodes/08-adding-ci-to-your-code.md

```diff
@@ -234,7 +234,7 @@ build_skim:
 
 # Building multiple versions
 
-Great, so we finally got it working... CI/CD isn't obviously powerful when you're only building one thing. Let's build both the version of the code we're testing and also test that the latest ROOT image (`rootproject/root-conda:latest`) works with our code. Call this new job `build_skim_latest`.
+Great, so we finally got it working... CI/CD isn't obviously powerful when you're only building one thing. Let's build both the version of the code we're testing and also test that the latest ROOT image (`rootproject/root:latest`) works with our code. Call this new job `build_skim_latest`.
```

Comment:
> CI/CD isn't obviously powerful when you're only building one thing.

Not sure how much sense this sentence makes to me, I think we can just delete it

Reply:
Agreed